cmd/collector/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM centos:7
```

Comment:
We could, as long as the contents for the Docker images are in sub directories from where the Dockerfiles are located. I'd rather keep it alongside with the image's contents.